Tools/TestWebKitAPI/cocoa/TestWKWebView.mm

```diff
@@ -760,6 +766,34 @@ - (void)evaluateJavaScriptAndWaitForInputSessionToChange:(NSString *)script
     }
 }
 
+- (CGFloat)overrideTopSafeAreaInset
+{
+    return _overrideTopSafeAreaInset;
+}
+
+- (void)setOverrideTopSafeAreaInset:(CGFloat)value
+{
+    _overrideTopSafeAreaInset = value;
+}
+
+- (CGFloat)overrideBottomSafeAreaInset
+{
+    return _overrideBottomSafeAreaInset;
+}
+
+- (void)setOverrideBottomSafeAreaInset:(CGFloat)value
+{
+    _overrideBottomSafeAreaInset = value;
+}
+
+- (UIEdgeInsets)safeAreaInsets
+{
+    auto inset = super.safeAreaInsets;
+    inset.top = _overrideTopSafeAreaInset;
+    inset.bottom = _overrideBottomSafeAreaInset;
```

Comment:
If we're going to be overriding to zero already in most cases, should we just add a single `overrideSafeAreaInset` property, that is a `UIEdgeInsets`?

Reply:
Sounds good to me! (My original thought was that maybe we'd want to be able to easily fall back on the original safe area values; but this sort of defeats the purpose of ensuring API test consistency, so just having a single override for all `UIEdgeInsets` likely makes more sense).

Thanks for the review!